### PR TITLE
Bump deps and to 4.13.1-0

### PR DESCRIPTION
**Pull request by compulim**
> Related to #3826

### Changed

-  Bumped all dependencies to the latest versions and sample bumps, by [@compulim](https://github.com/compulim) in PR [#3831](https://github.com/microsoft/BotFramework-WebChat/pull/3831)
   -  Development dependencies
      -  [`@azure/storage-blob@12.5.0`](https://npmjs.com/package/@azure/storage-blob)
      -  [`@babel/cli@7.13.14`](https://npmjs.com/package/@babel/cli)
      -  [`@babel/core@7.13.14`](https://npmjs.com/package/@babel/core)
      -  [`@babel/plugin-proposal-async-generator-functions@7.13.8`](https://npmjs.com/package/@babel/plugin-proposal-async-generator-functions)
      -  [`@babel/plugin-proposal-class-properties@7.13.0`](https://npmjs.com/package/@babel/plugin-proposal-class-properties)
      -  [`@babel/plugin-proposal-object-rest-spread@7.13.8`](https://npmjs.com/package/@babel/plugin-proposal-object-rest-spread)
      -  [`@babel/plugin-transform-runtime@7.13.10`](https://npmjs.com/package/@babel/plugin-transform-runtime)
      -  [`@babel/preset-env@7.13.12`](https://npmjs.com/package/@babel/preset-env)
      -  [`@babel/preset-react@7.13.13`](https://npmjs.com/package/@babel/preset-react)
      -  [`@babel/preset-typescript@7.13.0`](https://npmjs.com/package/@babel/preset-typescript)
      -  [`@babel/runtime@7.13.10`](https://npmjs.com/package/@babel/runtime)
      -  [`@babel/standalone@7.13.14`](https://npmjs.com/package/@babel/standalone)
      -  [`@fluentui/react@8.9.0`](https://npmjs.com/package/@fluentui/react)
      -  [`@types/node@14.14.37`](https://npmjs.com/package/@types/node)
      -  [`@types/react@17.0.3`](https://npmjs.com/package/@types/react)
      -  [`@typescript-eslint/eslint-plugin@4.21.0`](https://npmjs.com/package/@typescript-eslint/eslint-plugin)
      -  [`@typescript-eslint/parser@4.21.0`](https://npmjs.com/package/@typescript-eslint/parser)
      -  [`abort-controller-es5@1.2.0`](https://npmjs.com/package/abort-controller-es5)
      -  [`abort-controller@3.0.0`](https://npmjs.com/package/abort-controller)
      -  [`babel-loader@8.2.2`](https://npmjs.com/package/babel-loader)
      -  [`base64-arraybuffer@0.2.0`](https://npmjs.com/package/base64-arraybuffer)
      -  [`botbuilder-dialogs@4.12.0`](https://npmjs.com/package/botbuilder-dialogs)
      -  [`botbuilder@4.12.0`](https://npmjs.com/package/botbuilder)
      -  [`classnames@2.3.1`](https://npmjs.com/package/classnames)
      -  [`concurrently@6.0.1`](https://npmjs.com/package/concurrently)
      -  [`copy-webpack-plugin@8.1.1`](https://npmjs.com/package/copy-webpack-plugin)
      -  [`core-js@3.10.0`](https://npmjs.com/package/core-js)
      -  [`cross-env@7.0.3`](https://npmjs.com/package/cross-env)
      -  [`css-loader@5.2.0`](https://npmjs.com/package/css-loader)
      -  [`dotenv@8.2.0`](https://npmjs.com/package/dotenv)
      -  [`eslint-config-standard@16.0.2`](https://npmjs.com/package/eslint-config-standard)
      -  [`eslint-plugin-import@2.22.1`](https://npmjs.com/package/eslint-plugin-import)
      -  [`eslint-plugin-node@11.1.0`](https://npmjs.com/package/eslint-plugin-node)
      -  [`eslint-plugin-promise@4.3.1`](https://npmjs.com/package/eslint-plugin-promise)
      -  [`eslint-plugin-react@7.23.1`](https://npmjs.com/package/eslint-plugin-react)
      -  [`eslint-plugin-react@7.23.1`](https://npmjs.com/package/eslint-plugin-react)
      -  [`eslint-plugin-standard@5.0.0`](https://npmjs.com/package/eslint-plugin-standard)
      -  [`eslint@7.23.0`](https://npmjs.com/package/eslint)
      -  [`event-iterator@2.0.0`](https://npmjs.com/package/event-iterator)
      -  [`expect@26.6.2`](https://npmjs.com/package/expect)
      -  [`global-agent@2.2.0`](https://npmjs.com/package/global-agent)
      -  [`html-webpack-plugin@5.3.1`](https://npmjs.com/package/html-webpack-plugin)
      -  [`http-proxy-middleware@1.1.0`](https://npmjs.com/package/http-proxy-middleware)
      -  [`husky@6.0.0`](https://npmjs.com/package/husky)
      -  [`jest-image-snapshot@4.4.1`](https://npmjs.com/package/jest-image-snapshot)
      -  [`lerna@4.0.0`](https://npmjs.com/package/lerna)
      -  [`lint-staged@10.5.4`](https://npmjs.com/package/lint-staged)
      -  [`lolex@6.0.0`](https://npmjs.com/package/lolex)
      -  [`math-random@2.0.1`](https://npmjs.com/package/math-random)
      -  [`msal@1.4.9`](https://npmjs.com/package/msal)
      -  [`node-dev@6.6.0`](https://npmjs.com/package/node-dev)
      -  [`nodemon@2.0.7`](https://npmjs.com/package/nodemon)
      -  [`office-ui-fabric-react@7.165.2`](https://npmjs.com/package/office-ui-fabric-react)
      -  [`p-defer-es5@1.2.1`](https://npmjs.com/package/p-defer-es5)
      -  [`p-defer@3.0.0`](https://npmjs.com/package/p-defer)
      -  [`prettier@2.2.1`](https://npmjs.com/package/prettier)
      -  [`react-redux@7.2.3`](https://npmjs.com/package/react-redux)
      -  [`react-scripts@4.0.3`](https://npmjs.com/package/react-scripts)
      -  [`redux@4.0.5`](https://npmjs.com/package/redux)
      -  [`restify@8.5.1`](https://npmjs.com/package/restify)
      -  [`selenium-webdriver@4.0.0-beta.2`](https://npmjs.com/package/selenium-webdriver)
      -  [`simple-update-in@2.2.0`](https://npmjs.com/package/simple-update-in)
      -  [`source-map-loader@1.1.3`](https://npmjs.com/package/source-map-loader)
      -  [`string-similarity@4.0.4`](https://npmjs.com/package/string-similarity)
      -  [`style-loader@2.0.0`](https://npmjs.com/package/style-loader)
      -  [`uuid@8.3.2`](https://npmjs.com/package/uuid)
      -  [`webpack-cli@4.6.0`](https://npmjs.com/package/webpack-cli)
      -  [`webpack-stats-plugin@1.0.3`](https://npmjs.com/package/webpack-stats-plugin)
      -  [`webpack@4.46.0`](https://npmjs.com/package/webpack)
   -  Production dependencies
      -  [`classnames@2.3.1`](https://npmjs.com/package/classnames)
      -  [`core-js@3.10.0`](https://npmjs.com/package/core-js)
      -  [`create-emotion@11.0.0`](https://npmjs.com/package/create-emotion)
      -  [`react-redux@7.2.3`](https://npmjs.com/package/react-redux)
      -  [`redux-devtools-extension@2.13.9`](https://npmjs.com/package/redux-devtools-extension)
      -  [`url-search-params-polyfill@8.1.1`](https://npmjs.com/package/url-search-params-polyfill)
      -  [`whatwg-fetch@3.6.2`](https://npmjs.com/package/whatwg-fetch)

## Description

Bump to `4.13.1-0` and bump dependencies.

## Specific Changes

- Bump deps on `**/package.json`

-  [ ] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [x] I have updated documentation

## Review Checklist

> This section is for contributors to review your work.

-  [x] ~Accessibility reviewed (tab order, content readability, alt text, color contrast)~
-  [x] ~Browser and platform compatibilities reviewed~
-  [x] ~CSS styles reviewed (minimal rules, no `z-index`)~
-  [x] ~Documents reviewed (docs, samples, live demo)~
-  [x] ~Internationalization reviewed (strings, unit formatting)~
-  [x] `package.json` and `package-lock.json` reviewed
-  [x] ~Security reviewed (no data URIs, check for nonce leak)~
-  [x] ~Tests reviewed (coverage, legitimacy)~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,36 +37,36 @@
...
+    "lint-staged": "^10.5.4",
     "lolex": "^6.0.0",
     "node-fetch": "^2.6.1",
     "p-defer": "^3.0.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "selenium-webdriver": "^4.0.0-alpha.7",
+    "selenium-webdriver": "^4.0.0-beta.2",
```
Comment (corinagum) [inline package.json:69]: Any status on changing to version 89?

Comment (corinagum) [reply package.json:69]: Can you leave a comment on #3826?

**Review by corinagum — APPROVED** [review]
LGTM pending one comment

File: package.json (hunk 1 above)
Comment (compulim) [reply package.json:69]: Will do the Docker image in another PR, it may affect our screenshots.

**Pull request merged**